src/ImportDefinitionsBundle/ProcessManager/ProcessManagerListener.php

```diff
@@ -73,5 +79,46 @@ public function onStatusEvent(ImportDefinitionEvent $event)
             $this->process->setMessage($event->getSubject());
             $this->process->save();
         }
+        $this->logEvent('import_definition.status', $event->getSubject());
+    }
+
+    /**
+     * @param ImportDefinitionEvent $event
+     */
+    public function onFinishedEvent(ImportDefinitionEvent $event)
+    {
+        $this->logEvent('import_definition.finished', $event->getSubject());
+    }
+
+    /**
+     * @param ImportDefinitionEvent $event
+     */
+    public function onObjectStartEvent(ImportDefinitionEvent $event)
+    {
+        $this->logEvent('import_definition.object.start', $event->getSubject());
+    }
+
+    /**
+     * @param ImportDefinitionEvent $event
+     */
+    public function onObjectFinishedEvent(ImportDefinitionEvent $event)
+    {
+        $this->logEvent('import_definition.object.finished', $event->getSubject());
+    }
+
+    /**
+     * @param string $eventName
+     * @param string $message
+     */
+    protected function logEvent(string $eventName, string $message)
+    {
+        if (!$this->process) {
+            return;
+        }
+
+        if (!is_dir(PIMCORE_LOG_DIRECTORY . '/imports/')) {
+            mkdir(PIMCORE_LOG_DIRECTORY . '/imports/');
+        }
+        Simple::log('imports/process_' . $this->process->getId(), $eventName . ': ' . $message);
```

Comment:
not sure if it is a good idea to create a new log file for every process here. I think process manager already does that as well, doesn't it?

I would be more confident to add that to the database directly, within the `Process` itself. So we only need to clear one table in order to get rid of unneeded data.